pkg/strc/middleware.go

```diff
@@ -36,33 +36,9 @@ type MiddlewareConfig struct {
 	// ServerErrorLevel is the log level for requests with server errors (5xx). Defaults to Error.
 	ServerErrorLevel slog.Level
 
-	// SpanName is the name of the span. Defaults to "http request".
-	SpanName string
-
-	// WithUserAgent enables logging of the User-Agent header. Defaults to false.
-	WithUserAgent bool
-
-	// WithRequestBody enables logging of the request body. Defaults to false.
-	WithRequestBody bool
-
-	// WithRequestHeader enables logging of the request headers. Defaults to false.
-	WithRequestHeader bool
-
 	// WithResponseBody enables logging of the response body. Defaults to false.
 	WithResponseBody bool
```

Comment:
Please remove this as well plus all the infrastructure that is needed for buffering body. It is just too much code and i would rather simplify this quite a bit.